router_client.go

```diff
@@ -999,3 +1000,35 @@ func (r *routerClient) ResetMissionControl(ctx context.Context) error {
 	)
 	return err
 }
+
+type ChannelStatusAction int32
+
+const (
+	ChanStatusActionENABLE  ChannelStatusAction = 0
+	ChanStatusActionDISABLE ChannelStatusAction = 1
+	ChanStatusActionAUTO    ChannelStatusAction = 2
+)
+
+// UpdateChanStatus attempts to manually set the state of a channel (enabled,
+// disabled, or auto). A manual "disable" request will cause the channel to
+// stay disabled until a subsequent manual request of either "enable" or "auto".
+func (r *routerClient) UpdateChannelStatus(ctx context.Context,
```

Comment:
I'd name this function `UpdateChanStatus` to make it work with the macaroon black magic in `macaroon_recipes.go`. Or alternatively you can add a rename mapping there (in which case godoc needs to be updated at line 1012).